roles/ceph-common/tasks/main.yml

```diff
@@ -236,6 +236,11 @@
 # The following commit https://github.com/ceph/ceph/commit/791eba81a5467dd5de4f1680ed0deb647eb3fb8b
 # fixed a package issue where the path was the wrong.
 # This bug is not yet on all the distros package so we are working around it
+# Impacted versions:
+# - Jewel from UCA: https://bugs.launchpad.net/ubuntu/+source/ceph/+bug/1582773
+# - Jewel from latest Canonical 16.04 distro
+# - All previous versions from Canonical
```

Comment:
What does "All previous versions from Canonical" mean?

If I was maintaining this Ansible code a year from now and I read that comment, the phrasing would make it unclear if we can ever revert this.